staging/src/k8s.io/api/networking/v1/types_swagger_doc_generated.go

```diff
@@ -238,7 +238,7 @@ func (NetworkPolicyPort) SwaggerDoc() map[string]string {
 }
 
 var map_NetworkPolicySpec = map[string]string{
-	"":            "NetworkPolicySpec provides the specification of a NetworkPolicy",
+	"":            "NetworkPolicySpec provides the specification of a NetworkPolicy.  A suite of network policies allow traffic ONLY if BOTH the egress AND the ingress are allowed.  For example, if ingress is allowed by egress is not, or egress is allowed by ingress is not, the traffic will not be allowed.  See the Ingress and Egress fields' documentation for more information on when ingress and egress are allowed.",
```

Comment:
same comment as earlier:

if ingress is allowed **but** egress is not, or egress is allowed **but** ingress is not...

Reply:
👍